### ISSUE TITLE
Model "deberta-v2--xxlarge-mnli" doesn't work!!!

### ISSUE THREAD

**Issue by ngoquanghuy99** [huggingface/transformers#10625]
Whenever i try to load the tokenizer by
```python
from transformers import AutoTokenizer, AutoModel
tokenizer = AutoTokenizer.from_pretrained('microsoft/deberta-v2-xxlarge-mnli')
```
it returns this issue:
```
config_class = CONFIG_MAPPING[config_dict["model_type"]]
KeyError: 'deberta-v2'
```

Is this model not available in Transformers 4.3.3 (the latest release)?

**Comment by LysandreJik** [comment]
No, DeBERTa-v2 is not available in v4.3.3, it's only available from source as of now. Version v4.4.0 should be released end of this week or early next week, and will have DeBERTa-v2.

**Comment by ngoquanghuy99** [comment]
Yes, exactly what i think! Thanks Lysandre for confirming this.
